app.kaiteki.Kaiteki.yml

```
@@ -1,17 +1,41 @@
 app-id: app.kaiteki.Kaiteki
-runtime: org.gnome.Platform
-runtime-version: "45"
-sdk: org.gnome.Sdk
+runtime: org.freedesktop.Platform
+runtime-version: "22.08"
+sdk: org.freedesktop.Sdk
 command: app.kaiteki.Kaiteki
 separate-locales: false
 finish-args:
   - --device=dri
   - --share=ipc
   - --share=network
   - --socket=pulseaudio
-  - --socket=fallback-x11
-  - --socket=wayland
+  - --socket=x11
+cleanup:
+  - .la
+  - '*.a'
+  - /cache
+  - /lib/valadoc
+  - /lib/valadoc-0.48
+  - /man
+  - /share/aclocal
+  - /share/devhelp
+  - /share/gir-1.0
+  - /share/gtk-doc
+  - /share/man
+  - /share/vala-0.40
+  - /share/vala-0.48
+  - /share/valadoc
+  - /share/valadoc-0.48
 modules:
+  - name: libhandy
+    buildsystem: meson
+    config-opts:
+      - -Dexamples=false
```

```suggestion
      - -Dexamples=false
      - -Dvapi=false
      - -Dtests=false
```
https://gitlab.gnome.org/GNOME/libhandy/-/blob/1.8.2/meson_options.txt